README.md

```diff
@@ -28,35 +28,6 @@ The code in this repository is released under the Apache License, see the [LICEN
 
 Please use [this guide](CONTRIBUTING.md) to make a contribution to the project and information to report any issues.
 
-## Development
```

Comment:
one last thing: restore this section with just a phrase and a link to the new page, so it's more easy to discover.